app/validators/interests_of_justice/answers_validator.rb

```diff
@@ -0,0 +1,37 @@
+module InterestsOfJustice
+  class AnswersValidator < BaseAnswerValidator
+    def initialize(crime_application)
+      record = crime_application.ioj || Ioj.new(case: crime_application.kase)
+
+      super(record:, crime_application:)
+    end
+
+    def applicable?
+      return false if crime_application.cifc?
+
+      true
+    end
+
+    def validate
+      return if complete?
+
+      errors.add(:ioj, :incomplete)
+      errors.add(:base, :incomplete_records)
+    end
+
+    def complete?
+      crime_application.ioj_passported? || ioj_answers_complete?
```

Comment:
Wondering whether `passported` is more of a not-applicable scenario - could it confuse a user to update to `complete` when they did not complete anything?

Reply:
Ideally, we want the task list's 'non-applicable' and the answer validator's 'applicable?' to be inverses of each other. For this to happen, 'applicable?' in the validator should function as the inverse of 'not_applicable?'.

For example, in a "change in financial circumstances" application, the 'Interests of Justice' section is not applicable. However, for an under-18 applicant, the IoJ is applicable to the application but passported, so it is automatically completed.

I think we're at the point where we can start defining  what 'applicable?' means in the context of an answer validator, and we now have enough of a pattern to consider a refactor and see if we can standardise each interface and clearly define their roles. Does that make sense?

Reply:
Yep, that makes sense.